extensions/transaction/transaction-atomikos/src/test/java/org/eclipse/dataspaceconnector/transaction/atomikos/AtomikosTransactionExtensionTest.java

```diff
@@ -48,6 +50,8 @@ void verifyEndToEndTransactions() {
         when(extensionContext.getConfig()).thenReturn(ConfigFactory.fromMap(Map.of(TransactionManagerConfigurationKeys.LOGGING, "false")));
```

Comment:
Why getConfig() response needs to be recorded twice?

Reply:
Could you perhaps add a note explaining this fix in the PR description?

Reply:
There are 2 invocations, one with a String argument and one without. This is adding mock behaviour for the second one.

Reply:
Improved PR description